deployments/mailService/mailService.test.js

```diff
@@ -0,0 +1,13 @@
+const { sendReportEmail } = require("./mailService");
+
+// You can run this with:  dappMailerUrl=givethDevMailList=email1,email2 dappMailerSecret=  node deployments/mailService/mailService.test.js
```

Comment:
@aminlatifi  I will send `dappMailerUrl` and `dappMailerSecret` for you in DM to can test this